_config.yml

```diff
@@ -0,0 +1,4 @@
+title                       : Jupyter Governance
+author                      : The Jupyter Book community
+copyright                   : "2020" 
```

Comment:
Hmmm...we're explicitly licensing this under CC0. Perhaps we still need to give a copyright date? If so, it would be better to use 2015, when it was created.

Reply:
Is there a way in Jupyter Book to give a link to the license of the book itself here? For example, instead of it rendering as "By Author, (c) Copyright 2015", have it say something like "By Author, (c) Copyright 2015, Distributed under the terms of the license", where it links to the book license? That would help clarify in the book itself what the license of the book was.

Reply:
Ah interesting - right now, no, but I think that is a good idea to add a link for the license. Will add that to the to-do list for jupyter book

Reply:
Minimally, I think this should change to 2015.

```suggestion
copyright                   : "2015" 
```

Reply:
on the other hand, can you just delete the copyright year and have it not mention copyright?

Reply:
yep that works - I'm also adding the ability for us to add arbitrary HTML in the footer in case we'd like a link or something : https://github.com/executablebooks/sphinx-book-theme/pull/99

Reply:
I just added a link to the license in the footer: https://8-41781271-gh.circle-artifacts.com/0/html/README.html  what do you think?

Reply:
I see 
<img width="304" alt="Screen Shot 2020-05-20 at 11 23 45 AM" src="https://user-images.githubusercontent.com/192614/82482908-680cf500-9a8c-11ea-8f42-fd799f082a23.png">

I think for copyright you have to state the owner as well: `(c) Copyright 2015 Project Jupyter`.

Going through the CC0 attribution wizard (and slightly modifying the links in it), we get this HTML:

```html
<p xmlns:dct="http://purl.org/dc/terms/" xmlns:vcard="http://www.w3.org/2001/vcard-rdf/3.0#">
  <a rel="license"
     href="http://creativecommons.org/publicdomain/zero/1.0/">
    <img src="https://licensebuttons.net/p/zero/1.0/88x31.png" style="border-style: none;" alt="CC0" />
  </a>
  <br />
  To the extent possible under law,
  <a rel="dct:publisher"
     href="https://jupyter.org">
    <span property="dct:title">Project Jupyter</span></a>
  has waived all copyright and related or neighboring rights to
  <a href="https://github.com/jupyter/governance"><span property="dct:title">Project Jupyter Governance Documents</span></a>.
This work is published from:
<span property="vcard:Country" datatype="dct:ISO3166"
      content="US" about="https://jupyter.org">
  United States</span>.
</p>
```
which looks like 
<p xmlns:dct="http://purl.org/dc/terms/" xmlns:vcard="http://www.w3.org/2001/vcard-rdf/3.0#">
  <a rel="license"
     href="http://creativecommons.org/publicdomain/zero/1.0/">
    <img src="https://licensebuttons.net/p/zero/1.0/88x31.png" style="border-style: none;" alt="CC0" />
  </a>
  <br />
  To the extent possible under law,
  <a rel="dct:publisher"
     href="https://jupyter.org">
    <span property="dct:title">Project Jupyter</span></a>
  has waived all copyright and related or neighboring rights to
  <a href="https://github.com/jupyter/governance"><span property="dct:title">Project Jupyter Governance Documents</span></a>.
This work is published from:
<span property="vcard:Country" datatype="dct:ISO3166"
      content="US" about="https://jupyter.org">
  United States</span>.
</p>

Reply:
oooh I love that - I'll insert that in there.

I wonder if we could programmatically generate that with Jupyter Book in the future...you've given me more ideas!